package.json

```diff
@@ -22,7 +22,6 @@
     "test:cov": "NODE_ENV=test jest --collect-coverage --watchAll"
   },
   "engines": {
-    "node": "12.x",
```

Comment:
Still need this for Heroku (I'm worried if we don't have this here we'll mistakenly use a newer version like v14, which could come with potential issues).

Reply:
Added back in